R/utils-json.R

```diff
@@ -196,7 +194,7 @@ getApprovals <- function(data, chain_nm, legend_nm, appr_var_all, plot_type=NULL
       d <- date_index_list[[i]]
       
       if(isDV){
-        applicable_dates <- points[['time']][d] - hours(12)
+        applicable_dates <- points[['time']][d]
```

Comment:
we should pull this line out of the if-else since it happens for both DV and not DV now